src/os400qc3.c

```diff
@@ -1104,14 +1074,28 @@ int _libssh2_hmac_sha512_init(libssh2_hmac_ctx *ctx,
 int _libssh2_hmac_update(libssh2_hmac_ctx *ctx,
                          const void *data, size_t datalen)
 {
-    return libssh2_os400qc3_hmac_update(ctx,                            \
-                                        data, datalen);
+    char dummy[64];
+    Qus_EC_t errcode;
```

Comment:
```
/home/MONNERAT/libssh2/src/os400qc3.c, 1082.38: CZM0045(30) Undeclared identifier len.
```
May be insert `int len = datalen;` here.

Reply:
Synced the name in the function body. I hope it's OK.